test/inductor/test_compiled_autograd.py

```diff
@@ -3049,13 +3054,14 @@ def test_cudagraphs_cpu_graph(self):
 
         self.assertEqual(counters["inductor"]["cudagraph_skips"], 1)
 
-    @requires_cuda_and_triton
+    @skipIfXpu(msg="AssertionError: Scalars are not equal!")
```

Comment:
This is a cudagraph case which xpu is not supported, please update the skip message. or just use requires_cuda_and_triton.